app/src/main/res/values/strings.xml

```diff
@@ -617,7 +617,11 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_railway_crossing_barrier_none">It\'s not</string>
     <string name="action_deselect_all">Deselect all</string>
     <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, …</string>
-
+    <string name="quest_maxweight_title">"What is the weight limit here?"</string>
+    <string name="quest_maxweight_answer_noSign">There is no sign…</string>
+    <string name="quest_maxweight_unusualInput_confirmation_description">This weight looks implausible. Are you sure that it is correct?</string>
+    <string name="quest_maxweight_answer_other_sign">There is an other weight limit sign</string>
```

Comment:
"an other" is likely to be misread as "another" instead of an *other*. Some suggested alternatives for answer_other_sign

* There is a complex weight limit sign
* There is a non-simple weight limit sign
* The sign has more than one weight limit
* The signage has more than one weight limit
* The signs have more than one weight limit

For the leave note text, I suggest the following change:

`-` Would you like to leave a note? Note with a photo will allow other mappers to handle it. Common signs will be also added to StreetComplete, allowing a direct input.
`+` Would you like to leave a note? A note with a photo will enable other mappers to handle it. Common signs will also be added to StreetComplete, allowing a direct input.

Also, I'm guessing the `…` should be on answer_other_sign, not on answer_noSign.

Reply:
Again, thanks for review of English!

> "an other" is likely to be misread as "another" instead of an other. Some suggested alternatives for answer_other_sign

As the next stage I am planning to add support for common complex signs, like R12-5 ( https://wiki.openstreetmap.org/wiki/File:MUTCD_R12-5.svg https://wiki.openstreetmap.org/wiki/Key:maxweight#United_States ).

So plan is that also some complex signs will be answerable directly within app.

So it would be nice to have something that would work both now and also after this planned change.

Reply:
> For the leave note text, I suggest the following change: 

Changed.

> Also, I'm guessing the … should be on answer_other_sign, not on answer_noSign.

Changed.

Reply:
In anticipation of complex signs, I suggest changing `an other` to [`some other`](https://www.theatlantic.com/politics/archive/2016/08/the-rise-of-the-others/497690/). It might look odd and ill-fitting and imprecise, but that's english. :)

`some other type of` would spell it out more clearly, if there's adequate room for the text. `some other kind of` could also work, but `kind` is more often used for broad ranges of animals/plants/materials, not as often for specialized subsets.

Reply:
> if there's adequate room for the text.

There is no more space, hopefully "some other" will fit.

Reply:
([The hyperlink](https://www.theatlantic.com/politics/archive/2016/08/the-rise-of-the-others/497690/) is only intended to demonstrate that `some other` is common/understood enough to appear on a US census form that a hundred million people will see.)

Reply:
see also https://github.com/westnordost/StreetComplete/pull/1467#discussion_r300831197